_posts/2021-05-01-sr2021-knockouts.md

```diff
@@ -0,0 +1,172 @@
+that previous years’, the core challenge to create an autonomous robot remained
+the same.
+
+{% include figure.html
+           class="right"
+           link_to_image="true"
+           src="/images/content/news/sr2021/league-4-top-teams.png"
+           caption="Teams’ game points per league session." %}
+
+During the league sessions there were some teams which showed strong early
+performances. Initially the teams from Hills Road Sixth Form College, looked to
+be the teams to beat. The later leagues however were the battleground of a
+different collection of teams, with KEGS Chelmsford (KEG), Gymnasium Markt
+Indersdorf (MAI) as well as rookie teams St Paul’s College (SPA) and We Robot
+(WER) in particular performing the best in the fourth and most complex of the
+leauge sessions.
+
+The Final
+---------
+
+The final match saw SPA and HRS3 battle for the top spot. Both teams having
+shown their incredible abilities throughout the competition. Whilst the robots
+started off with very different tactics, they were both accruing points very
+quickly. Due to a large number of link breakages, the points and leader switched
+massively throughout the match, but when the clock struck zero, it was 36 points
+to 8 for St Pauls, giving them the title.
+
+<!-- Video of Final -->
+<iframe
+  class="center video"
+  src="https://www.youtube-nocookie.com/embed/Jwk2kUp02lk"
+  frameborder="0"
+  width="100%"
+  allowfullscreen
+  >
+</iframe>
+
+If you didn’t take part, or you want to enter again next year, the
+[sign up page]({{ site.baseurl }}/compete/) for next year’s competition will be
+up later in the year. Get a team together and start talking to your teachers now!
+
+Results
+-------
+
+{% include figure.html
+           class="right"
+           src="/images/content/news/sr2021/trophy-render.png"
+           alt="A podium, with rendered trophies, featuring SPA in Gold, HRS2 in Silver, and MCK in Bronze"
+           caption="The podium positions" %}
+
+|        Prize          |            Team
+|-----------------------|-----------------------------------------------
+| 1<sup>st</sup> Place  | [SPA: St Pauls College, Adelaide](https://www.stpauls.sa.edu.au/)
+| 2<sup>nd</sup> Place  | [Hills Road Sixth Form College](https://www.hillsroad.ac.uk) Team 3
+| 3<sup>rd</sup> Place  | [The Malay College Kuala Kangsar](https://mckk.edu.my)
+| <span class="has-explanation" title="The team that displays the most extraordinary ingenuity or elegant simplicity in the design of their robot">Committee Award</span> | [Hills Road Sixth Form College](https://www.hillsroad.ac.uk) Team 2
+| <span class="has-explanation" title="The rookie team who achieve the highest place in the league">Rookie Award</span> | [We Robot](https://www.werobot.fr)
+| <span class="has-explanation" title="The team that is judged to have the best online presence">Online Presence</span> | [YSH: Yayasan Sultan Haji Hassanal Bolkiah](https://www.secondary-yayasan.edu.bn/en/)
```

Comment:
Use link sourced from team

```suggestion
| <span class="has-explanation" title="The team that is judged to have the best online presence">Online Presence</span> | [YSH: Yayasan Sultan Haji Hassanal Bolkiah](https://www.yshhb.org.bn/)
```

Reply:
For some reason GitHub wouldn't let me provide this via the UI: 8d4c2bf